reactor-core/src/test/java/reactor/core/publisher/UnicastProcessorTest.java

```diff
@@ -65,20 +65,19 @@ public void secondSubscriberRejectedProperly() {
 
 	@Test
 	public void multiThreadedProducer() {
-		UnicastProcessor<Integer> processor = UnicastProcessor.create();
-		FluxSink<Integer> sink = processor.sink();
+		Sink<Integer> sink = Sinks.unicast().onBackpressureBuffer();
```

Comment:
I'm not sure `xxxProcessor` tests should use the `Sinks` path. After all, these classes are there to explicitly test a specific concrete implementation and would likely survive the switch of said concrete implementation to package-private. On the other hand, if the implementation of `Sinks.unicast()` somehow changes then coverage would drop for `UnicastProcessor`.

This comment applies to other `xxxProcessorTest` cases.